sklearn/linear_model/logistic.py

```diff
@@ -1456,10 +1454,8 @@ def fit(self, X, y):
                 # Take the best scores across every fold and the average of all
                 # coefficients corresponding to the best scores.
                 best_indices = np.argmax(scores, axis=1)
-                w = np.mean([
-                    coefs_paths[i][best_indices[i]]
-                    for i in range(len(folds))
-                    ], axis=0)
+                w = np.mean([coefs_paths[i][best_indices[i]] for i in
```

Comment:
I'm not sure I like the line break after `in`, shouldn't we break at the `for` usually?